server/src/main/java/com/objectcomputing/checkins/services/role/RoleController.java

```diff
@@ -60,16 +88,19 @@
      * @param role, {@link Role}
      * @return {@link HttpResponse<Role>}
      */
-    @Put()
+    @Put("/")
     @Secured(RoleType.Constants.ADMIN_ROLE)
-    public HttpResponse<?> update(@Body @Valid Role role, HttpRequest<Role> request) {
-        Role updatedRole = roleServices.update(role);
-        return HttpResponse
-                .ok()
-                .headers(headers -> headers.location(
-                        URI.create(String.format("%s/%s", request.getPath(), updatedRole.getId()))))
-                .body(updatedRole);
-
+    public Single<HttpResponse<Role>> update(@Body @Valid Role role, HttpRequest<Role> request) {
+        if (role == null) {
+            return Single.just(HttpResponse.ok());
```

Comment:
Can `Role` be null if we use `@valid` annotation?

Reply:
Added @NotNull annotation with @Valid notation and removed unnecessary if null check.